chia/cmds/farm_funcs.py

```diff
@@ -211,26 +231,30 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
         print("Block rewards: Unknown")
         print("Last height farmed: Unknown")
 
-    total_plot_size = 0
+    if harvester_info is not None:
+        print(f"Connected harvesters: {harvester_info['connected_harvesters']}")
```

Comment:
```suggestion
        print(f"Total connected harvesters: {harvester_info['connected_harvesters']}")
```

make the log messages for both cases match